airflow/executors/base_executor.py

```diff
@@ -69,16 +70,16 @@ def start(self):  # pragma: no cover
         """
 
     def queue_command(self,
-                      simple_task_instance: SimpleTaskInstance,
```

Comment:
Should we be able to remove `SimpleTaskInstance` totally from Airflow?

Reply:
Yes, and we're almost there with this PR, but not quite -- it's still used for sending messages/failure callback requests _down_ to the Parsing process.